From 805eb739e94e84ff3566283b9a89dcacc7b871e8 Mon Sep 17 00:00:00 2001
From: Julian Sparber <julian@sparber.net>
Date: Thu, 22 Oct 2020 12:33:58 +0200
Subject: [PATCH 6/6] in-app-notification: wrap text and add start/end margin

This makes sure that the in-app-notification fits also small window
sizes and eventually wraps the text to a new.
---
 ui/components-in-app-notification.ui | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/ui/components-in-app-notification.ui b/ui/components-in-app-notification.ui
index 54a1dafd..adbb3e2f 100644
--- a/ui/components-in-app-notification.ui
+++ b/ui/components-in-app-notification.ui
@@ -5,6 +5,8 @@
     <property name="visible">False</property>
     <property name="halign">center</property>
     <property name="valign">start</property>
+    <property name="margin-start">12</property>
+    <property name="margin-end">12</property>
     <signal name="notify::child-revealed" handler="on_child_revealed" swapped="no"/>
     <child>
       <object class="GtkBox" id="layout">
@@ -17,18 +19,21 @@
         <child>
           <object class="GtkLabel" id="message_label">
             <property name="visible">True</property>
+            <property name="wrap">True</property>
           </object>
         </child>
         <child>
           <object class="GtkButton" id="action_button">
             <property name="visible">False</property>
             <property name="can_focus">False</property>
+            <property name="valign">center</property>
           </object>
         </child>
         <child>
           <object class="GtkButton" id="close_button">
             <property name="visible">True</property>
             <property name="can_focus">False</property>
+            <property name="valign">center</property>
             <signal name="clicked" handler="close" swapped="no"/>
             <style>
               <class name="flat"/>
-- 
2.29.2

